src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/string.txt

```diff
@@ -350,7 +350,7 @@ Errors: Error 2-2: Unexpected characters. Characters are used in the formula in
 
 >> $"{
 /*SqlRunner*/ Compile Error
-Errors: Error 0-2: Invalid number of arguments: received 0, expected 1 or more.
+""
```

Comment:
Why is this case not erroneous? There's no matching `"`, shouldn't it be a parser error?

Reply:
Actually no, and functions work the same way. Intellisense still works with partial expressions like this, which is why it's not actually a parse error.